test/controllers/surveys_controller_test.rb

```diff
@@ -74,6 +76,7 @@ class SurveysControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should update survey" do
+    sign_in @user
     patch survey_url(@survey), params: { survey: { name: @survey.name, published: @survey.published, user_id: @survey.user_id } }
```

Comment:
Should update to what?

Reply:
Sorry @spencerldixon I haven't fixed all of the tests yet. I just pushed to make my commits more atomic.